cairo/kakarot-ssj/crates/contracts/src/token/erc20.cairo

```diff
@@ -0,0 +1,109 @@
+#[starknet::contract]
+pub mod ERC20 {
+    use openzeppelin::introspection::src5::SRC5Component;
+    use openzeppelin::token::erc20::{ERC20Component, ERC20HooksEmptyImpl};
+    use starknet::{ContractAddress, get_caller_address};
+
+    component!(path: ERC20Component, storage: erc20, event: ERC20Event);
+    component!(path: SRC5Component, storage: src5, event: SRC5Event);
```

Comment:
is SRC5 needed? I think we can do without ? This token is only used in tests anyway, to mock the ERC20 native_token